### PR TITLE
Added logout command and parser

### PR DESCRIPTION
**Pull request by IcedCoffeeBoy**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/InvalidLoginParser.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            + "\nPlease logout to login again"
+            + "\nExample: logout";
+
+    /**
+     * Parses user input into command for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public LoginCommand parse(String identity) throws ParseException {
+        throw new ParseException(String.format(MESSAGE_INVALID_LOGIN, identity));
```
Comment (jieliangang) [inline src/main/java/seedu/address/logic/parser/InvalidLoginParser.java]: why is there a separate parser class which first does not parse, and second only throws a ParseException. the throwing of ParseException can be added to the Manager/Employee parser without the need of a separate class. having the class called Parser can be misleading as it does not work as a parser.

File: src/main/java/seedu/address/logic/parser/InvalidPrivilegeParser.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            + "\nCurrent login as %s"
+            + "\nPlease login account with higher privilege to use this command";
+
+    /**
+     * Parses user input into command for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public LoginCommand parse(String identity) throws ParseException {
+        throw new ParseException(String.format(MESSAGE_INVALID_LOGIN, identity));
```
Comment (jieliangang) [inline src/main/java/seedu/address/logic/parser/InvalidPrivilegeParser.java]: similar to InvalidLoginParser. a class should not be created for a one line method which throws a parse exception.

File: src/main/java/seedu/address/logic/parser/InvalidLoginParser.java (hunk 1 above)
Comment (IcedCoffeeBoy) [reply src/main/java/seedu/address/logic/parser/InvalidLoginParser.java]: how you you suggest to do throw the exception without breaking the OOP?

Comment (IcedCoffeeBoy) [reply src/main/java/seedu/address/logic/parser/InvalidLoginParser.java]: i can throw exception directly

Comment (jieliangang) [reply src/main/java/seedu/address/logic/parser/InvalidLoginParser.java]: will throwing exception directly break OOP?

**Comment by IcedCoffeeBoy** [comment]
Ok i change the invalid parser class in invalid exception class

**Pull request merged**